src/Octoshift/Handlers/WaitForMigrationCommandHandler.cs

```diff
@@ -108,20 +108,22 @@ private async Task WaitForRepositoryMigration(string migrationId, string githubP
             if (RepositoryMigrationStatus.IsSucceeded(state))
             {
                 _log.LogSuccess($"Migration {migrationId} succeeded for {repositoryName}");
+                _log.LogInformation($"Migration log available at: {url}");
```

Comment:
CLI also has a `download-logs ` command that is used for downloading the migration log. Perhaps, in addition to spitting out the log url, we can also say something like this:
```
Migration log is available at : {url} or can be downloaded with `download-logs` command.
```
I guess @timrogers can help with the copy. 

Or maybe alternatively we can just say that the migration log can be downloaded with the `download-logs` command w/o even spitting out the url and going through the hassle of returning the `MigrationLogUrl` at all. This is something needs @timrogers input 😃

Reply:
I think it'd be a great idea to print out the URL, just because it makes it immediately accessible and will encourage people to check it out, rather than hiding it behind another command ❤️

Reply:
Here's my proposal:

```suggestion
                _log.LogInformation($"Migration log available at <{url}> or by running `gh gei download-logs`");
```

In the context of this method, do we know if it's `gei`, `bbs2gh` or `ado2gh`?

Reply:
> In the context of this method, do we know if it's gei, bbs2gh or ado2gh?
yes we can know. Simply by calling `Assembly.GetEntryAssembly()` and I think it's not susceptible to JIT inlining so we should be good. So the info log will be like this:
```suggestion
                _log.LogInformation($"Migration log available at {url} or by running `gh {Assembly.GetEntryAssembly()!.GetName().Name} download-logs`");
```
> It would be even nicer to print the full command, but to do that, we would need the target org and target API URL in this method.
Yes. Although possible, it would be too much of a hassle to print the full command.